doc/en/user/source/services/wms/webadmin.rst

```diff
@@ -46,6 +46,18 @@ The Web Map Service Interface Standard (WMS) provides a simple way to request an
 
 **Bicubic** -Looks at the sixteen nearest cells and fits a smooth curve through the points to find the output value. Bicubic interpolation may both change the input value as well as place the output value outside of the range of input values. Bicubic interpolation is recommended for smoothing continuous data, but this incurs a processing performance overhead.
 
+Dimensions settings
+-------------------
+
+The WMS specification mandates to throw an ``InvalidDimensionValue`` exception when a dimension value is not valid, and the nearest match is not enabled. 
```

Comment:
"The WMS specification mandates the throwing of an"  or "The WMS specification mandates that we throw an"